macro-rules/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+mod java_types;
```

Comment:
as a small point, it's hard to find where the macros are actually defined because they all get exported into the root namespace. I guess there's no way around that though :-/

Reply:
No, no way around that, but we could break them into a file per macro.